### DISCUSSION TITLE
How to access ownerRecord in table description

### DISCUSSION THREAD

**Discussion by gizzmojo** [filamentphp/filament#8938] [Help]
Inside a relationship manager table description how to access owner record?

Tried with this but getting error
```php
public function table(Table $table): Table
{
    return $table
        ->description(fn (ProductsRelationManager $livewire) => $livewire->ownerRecord->id)
```

> Typed property Filament\Resources\RelationManagers\RelationManager::$ownerRecord must not be accessed before initialization

**Comment by leandrocfe** [comment]
Not sure if this is the right way but you can do this

```php
public function render(): View
{
    $this->table->description($this->ownerRecord->id);
    return parent::render();
}
```

**Reply by gizzmojo** [reply]
It works but seems a little hacky doing it this way around when everything else can be done in the table method.
